test/node/users.js

```diff
@@ -24,6 +24,11 @@ describe('Users', () => {
         await client2.logout();
     });
 
+    it('should get the user', async () => {
+        const res = await client.getLoggedOnUser();
+        assert(res && res.userId === user.userId);
+    });
+    
     it('should get the logged on users', async () => {
```

Comment:
what is this test for? we already tested getLoggedOnUser.

this test should be for getUsersByEmail. So "should get users by their email"

Reply:
I think I did the same test twice by mistake. I removed the redundant one.